browser/brave_rewards/rewards_panel/rewards_panel_coordinator.cc

```diff
@@ -21,26 +18,12 @@ namespace brave_rewards {
 RewardsPanelCoordinator::RewardsPanelCoordinator(
     Browser* browser,
     RewardsService* rewards_service)
-    : BrowserUserData<RewardsPanelCoordinator>(*browser) {
-  // If we are using the Rewards extension to display the Rewards panel, then
-  // create an instance of `RewardsPanelExtensionHandler` to load the extension
-  // and dispatch panel requests to the extension.
-  if (!base::FeatureList::IsEnabled(features::kWebUIPanelFeature)) {
-    auto handler = std::make_unique<RewardsPanelExtensionHandler>(
-        &GetBrowser(), rewards_service);
-    AddObserver(handler.get());
-    extension_handler_ = std::move(handler);
```

Comment:
Do we still need `std::unique_ptr<Observer> extension_handler_;` in `rewards_panel_coordinator.h`?

Reply:
Good catch - thanks!